### ISSUE TITLE
Changing SigEp icon

### ISSUE THREAD

**Issue by BradHoffman96** [SigEpKYBeta/SigEp-WebApp#11]
Change the logo from the red text to this image http://www.sigep.org/resourcedocs/graphics/sigep_logotype_4cp.jpg

**Comment by SheldonB** [comment]
This might take a little tinkering. I played around with this before @BradHoffman96 added any content. I had some issues getting it to scale correctly.

**Comment by SheldonB** [comment]
I also need a logo with a transparent background if we leave the nav bar the way it is.

**Comment by SheldonB** [comment]
``` html
<li class="logo">
  <a id="logo-container" href="http://materializecss.com/" class="brand-logo">
    <object id="front-page-logo" type="image/svg+xml" data="res/materialize.svg">
       Your browser does not support SVG
    </object>
  </a>
</li>
```

I think something like this will work.

**Comment by SheldonB** [comment]
http://www.sigep.ga/assets/sigeplogo-a3dc626449ffc424e6ba707ab5bf0ab7.png

Found a png. Thank you Georgia Alpha!

**Comment by SheldonB** [comment]
Fixed in version 1.0.1
https://github.com/SheldonB/SigEp-WebApp/commit/f41491e9975425285265d52aee3ea4c8a9d36a18
